mysql/datadog_checks/mysql/innodb_metrics.py

```diff
@@ -0,0 +1,405 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import re
+from collections import defaultdict
+from contextlib import closing
+
+import pymysql
+from six import PY3, iteritems
+
+from datadog_checks.base import is_affirmative
+from datadog_checks.base.log import get_check_logger
+
+from .collection_utils import collect_scalar
+from .const import OPTIONAL_INNODB_VARS
+
+if PY3:
+    long = int
```

Comment:
Any idea why we need ? => 

```
if PY3:
    long = int
```

cc @ofek

Reply:
No, no idea